### DISCUSSION TITLE
Is this browser safe enough to store and manage my private data?

### DISCUSSION THREAD

**Discussion by zbouka** [zen-browser/desktop#3056] [Q&A]
So the thing is that i know this is a dumb question to ask, but i cant help but to say it anyways. I have been using this browser for a few days, and its has been delightful and refreshing to use in my day-to-day!
But i cant keep wondering if its safe for storing password, encrypted bank data among other things!
Thanks in advance

**Comment by theoneand33** [comment]
Well if Firefox is safe enough then Zen browser is because Zen browser is based off Firefox but for storing passwords I recommend Proton pass

**Comment by Evren-os** [comment]
For storing passwords, just use proton pass or bitwarden, don't rely on the browser's default manager. as for bank data, not sure. but since zen is firefox-based, and firefox is trusted and safe, zen should be good too

**Reply by theoneand33** [reply]
> For storing passwords, just use proton pass or bitwarden, don't rely on the browser's default manager. as for bank data, not sure. but since zen is firefox-based, and firefox is trusted and safe, zen should be good too

Thats what I said
